packages/adapter-netlify/src/edge.js

```diff
@@ -12,6 +12,7 @@ const prefix = `/${manifest.appDir}/`;
 export default function handler(request, context) {
 	if (is_static_file(request)) {
```

Comment:
Is it right to change this? `prefix` is used to determine `is_static_file`. This no longer seems correct to me or at least `is_static_file` is no longer named properly since things outside of `build` in `_app` like `version.json` are still static files

Reply:
ah yep, good catch. fixed